tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -304,39 +300,39 @@ def test_multiple_calls(self) -> None:
                 repo_trees=repo_trees,
                 frames=[self.frame("foo/bar.py", True)],
                 platform=platform,
-                expected_stack_root="foo/",
-                expected_source_root="src/foo/",
-                expected_repo_name=REPO1,
-                expected_num_code_mappings=1,
+                expected_new_code_mapping={
+                    "stack_root": "foo/",
+                    "source_root": "src/foo/",
+                    "repo_name": REPO1,
+                },
             )
-            # Processing the same stacktrace again should not create anything new
+            # Processing the same stacktrace again should not create anything new,
+            # thus, not passing in expected_new_code_mapping
             self._process_and_assert_configuration_changes(
-                repo_trees=repo_trees,
-                frames=[self.frame("foo/bar.py", True)],
-                platform=platform,
-                expected_stack_root="foo/",
```

Comment:
We now don't pass `expected` values in this second call since no new code mapping is expected.